x-pack/plugins/apm/server/types.ts

```diff
@@ -65,6 +65,10 @@ import {
   ProfilingDataAccessPluginSetup,
   ProfilingDataAccessPluginStart,
 } from '@kbn/profiling-data-access-plugin/server';
+import {
+  InternalElasticsearchServiceSetup,
+  InternalElasticsearchServiceStart,
+} from '@kbn/core-elasticsearch-server-internal';
```

Comment:
IMO, you should import from the non-internal package.
```suggestion
import {
  ElasticsearchServiceSetup,
  ElasticsearchServiceStart,
} from '@kbn/core-elasticsearch-server';
```

Reply:
Addressed in [8a2cad6](https://github.com/elastic/kibana/pull/166438/commits/8a2cad6c232b91ca73e4fabac8bf4614293e73be).